### PR TITLE
Added Star Lite Mk II and Star LabTop Mk III

### PR DESCRIPTION
**Pull request by Sean-StarLabs**

### CODE REVIEW

**Comment by yuwata** [comment]
CI fails. I guess unwanted white spaces in a blank line.
```
The output from the failed tests:
497/804 parse-hwdb                              FAIL     1.68 s (exit status 1)
--- command ---
/build/hwdb.d/parse_hwdb.py
--- stdout ---
Cannot parse /build/hwdb.d/60-evdev.hwdb: Expected stringEnd, found '#'  (at char 15715), (line:560, col:1)
/build/hwdb.d/60-evdev.hwdb: 0 match groups, 0 matches, 0 properties
/build/hwdb.d/60-evdev.hwdb: no matches or props
/build/hwdb.d/60-input-id.hwdb: 3 match groups, 3 matches, 4 properties
/build/hwdb.d/60-keyboard.hwdb: 174 match groups, 258 matches, 864 properties
/build/hwdb.d/60-sensor.hwdb: 108 match groups, 129 matches, 112 properties
/build/hwdb.d/70-joystick.hwdb: 2 match groups, 3 matches, 2 properties
/build/hwdb.d/70-mouse.hwdb: 107 match groups, 122 matches, 130 properties
/build/hwdb.d/70-pointingstick.hwdb: 8 match groups, 30 matches, 11 properties
/build/hwdb.d/70-touchpad.hwdb: 6 match groups, 9 matches, 6 properties
-------
```

File: hwdb.d/60-evdev.hwdb [hunk 1]
```diff
@@ -613,3 +631,4 @@ evdev:input:b0003v172Fp0031*
 evdev:input:b0003v172Fp0047*
  EVDEV_ABS_00=0:20000:80
  EVDEV_ABS_01=0:12500:80
+
```
Comment (yuwata) [inline hwdb.d/60-evdev.hwdb]: Please do not add this.

File: hwdb.d/60-evdev.hwdb [hunk 2]
```diff
@@ -563,6 +563,24 @@ evdev:name:ETPS/2 Elantech Touchpad:dmi:*svnSAMSUNGELECTRONICSCO.,LTD.:pn870Z5E/
  EVDEV_ABS_01=::29
  EVDEV_ABS_35=::30
  EVDEV_ABS_36=::29
+
```
Comment (yuwata) [inline hwdb.d/60-evdev.hwdb]: This line contains a white-space. Please drop it.

**Comment by whot** [comment]
Looks good otherwise, please fix the remaining whitespace issue and that should do the trick.

**Comment by keszybz** [comment]
@Sean-StarLabs ping.

**Pull request merged**